VectorSearchAiAssistant.SemanticKernel/Plugins/Core/TextSummaryPlugin.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.Connectors.AI.OpenAI;
+using Microsoft.SemanticKernel.Orchestration;
+using System.ComponentModel;
+
+namespace VectorSearchAiAssistant.SemanticKernel.Plugins.Core
+{
+    public class TextSummaryPlugin
+    {
+        private readonly ISKFunction _summarizeConversation;
+        private readonly IKernel _kernel;
+
+        public TextSummaryPlugin(
+            string promptTemplate,
+            int maxTokens,
+            IKernel kernel)
+        {
+            _kernel = kernel;
+            _summarizeConversation = kernel.CreateSemanticFunction(
+                promptTemplate,
+                pluginName: nameof(TextSummaryPlugin),
+                description: "Given a text, summarize the text.",
```

Comment:
Is the system prompt for this still coming from blob storage to drive the behavior?

Reply:
Yes, still coming from the blob storage account.